.github/workflows/build.yml

```diff
@@ -90,21 +85,19 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: ${{ matrix.php-version }}
+          php-version: 8.4
           extensions: grpc, protobuf
           tools: composer
 
       - name: Install dependencies
         run: composer install
 
-      - name: Set TEST_REDIS variable dynamically
-        run: echo "TEST_REDIS=$([ '${{ matrix.backend }}' == 'redis' ] && echo true || echo false)" >> $GITHUB_ENV
+      - name: Run tests with Momento backend
+        run: |
+          echo "Running tests with Momento backend"
+          composer test -- --group momento
 
-      - name: Run tests based on backend
+      - name: Run tests with Redis backend
         run: |
-          if [ "${{ matrix.backend }}" = "momento" ]; then
-            echo "Running tests with Momento backend"
-            composer test -- --group momento
-          else
-            echo "Running tests with Redis backend"
-            composer test -- --group redis
+          echo "Running tests with Redis backend"
+          composer test -- --group redis
```

Comment:
Do we not need to set the `TEST_REDIS` env var? Looking at actions output, no tests were run for either redis or momento.